test/Microsoft.AspNet.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -183,9 +213,16 @@ private static ActionContext GetActionContext(HttpContext httpContext)
                                     new ActionDescriptor());
         }
 
-        private static HttpContext GetHttpContext()
+        private static HttpContext GetHttpContext(bool addBufferingFeature = false)
```

Comment:
Another nitpick, sorry for not noticing. You might want to just put this inline in your tests that use the feature. That makes it much more apparent what you're doing.